File: other/sorts/bubble.rb
def bs array
    array.count.times do
        array.each_index do |i|
            if foo = array[i + 1]
                bar = array[i]
                if foo < bar
                    array[i] = foo
                    array[i + 1] = bar
                end
            end
        end
    end
    array
end

File: other/sorts/bubble.js
function bubble(array){
  for (var x = 0; x < array.length; x++){
    for (var y = 0; y < array.length - 1; y++){
      var foo = array[y]
      var bar = array[y + 1]
      if (foo > bar) {
        array[y] = bar
        array[y + 1] = foo
      }
    }
  }
  return array
}
